Content.Shared/Anomaly/Components/InnerBodyAnomalyComponent.cs

```diff
@@ -70,3 +70,12 @@ public sealed partial class InnerBodyAnomalyComponent : Component
     [DataField]
     public string LayerMap = "inner_anomaly_layer";
 }
+
+/// <summary>
+/// Event broadcast when an anomaly is being removed because the host is dying.
+/// Raised directed at the host entity with the anomaly.
+/// Cancel this if you want to prevent the host from losing their anomaly on death.
+/// </summary>
+public sealed class BeforeRemoveAnomalyOnDeathEvent : CancellableEntityEventArgs
+{
+}
```

Comment:
```suggestion
[ByRefEvent]
public record struct BeforeRemoveAnomalyOnDeathEvent(bool Cancelled = false);
```
Use a record struct instead of a class. A lot of old code is still using `CancellableEntityEventArgs`, but this is more efficient.

Reply:
I was spreading misinformation sorry 😢